packages/manager/src/views/SLA/SLAPage.tsx

```diff
@@ -0,0 +1,155 @@
+import { Banner } from '@app/components';
+import { PageSection } from '@patternfly/react-core';
+import { TableComposable, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
+
+export const SLAPage = (): JSX.Element => (
+  <>
+    <Banner title="Service Level Agreement for Spaship" />
```

Comment:
Please convert **Spaship** to **SPAship**

Reply:
```suggestion
    <Banner title="Service Level Agreement for SPAship" />
```

Reply:
good observation @soumyadip007

Reply:
Thanks @arkaprovob